releases/release-6.3.0.md

```diff
@@ -0,0 +1,303 @@
+---
+title: TiDB 6.3.0 Release Notes
+---
+
+# TiDB v6.3.0 Release Notes
+
+Release date: xx xx, 2022
+
+TiDB version: 6.3.0-DMR
+
+In v6.3.0-DMR, the key new features and improvements are as follows:
+
+- TiKV and TiFlash support SM4 encryption at rest.
+- TiDB supports authentication with the SM3 algorithm.
+- The `CREATE USER` and `ALTER USER` statements support the `ACCOUNT LOCK/UNLOCK` option.
+- JSON data type and functions become generally available (GA).
+- TiDB supports Null Aware Anti Join.
+- TiDB provides execution time metrics at a finer granularity.
+- A new syntactic sugar is added to simplify Range partition definitions.
+- Range Columns partitioning supports defining multiple columns.
+
+## New features
+
+### SQL
+
+* Add a new syntactic sugar (Range INTERVAL partitioning) to simplify Range partition definitions [#35683](https://github.com/pingcap/tidb/issues/35683) @[mjonss](https://github.com/mjonss)
+
+    [Provides INTERVAL partitioning as a new way of defining Range partitions](/partitioned-table.md#range-interval-partitioning). You do not need to enumerate all partitions, which drastically reduces the lengthy way of writing Range partition statements. The semantic is equivalent to the original Range partition.
+
+* Range Columns partitioning supports defining multiple columns [#36636](https://github.com/pingcap/tidb/issues/36636) @[mjonss](https://github.com/mjonss)
+
+    Support [PARTITION BY RANGE COLUMNS (column_list)](/partitioned-table.md#range-columns-partitioning). `column_list` is no longer limited to a single column. The basic feature is the same as MySQL.
+
+* EXCHANGE PARTITION becomes GA [#35996](https://github.com/pingcap/tidb/issues/35996) @[ymkzpx](https://github.com/ymkzpx)
+
+    [EXCHANGE PARTITION](/partitioned-table.md#partition-management) becomes GA after performance and stability improvements.
+
+* TiDB supports two more [window functions](/tiflash/tiflash-supported-pushdown-calculations.md) [#5579](https://github.com/pingcap/tiflash/issues/5579) @[SeaRise](https://github.com/SeaRise) **tw：shichun-0415**
+
+    * `LEAD`
+    * `LAG`
+
+* The `CREATE USER` statement supports the `ACCOUNT LOCK/UNLOCK` option [#37051](https://github.com/pingcap/tidb/issues/37051) @[CbcWestwolf](https://github.com/CbcWestwolf) **tw：ran-huang**
```

Comment:
```suggestion
* The `CREATE USER` and `ALTER USER` statements support the `ACCOUNT LOCK/UNLOCK` option [#37051](https://github.com/pingcap/tidb/issues/37051) @[CbcWestwolf](https://github.com/CbcWestwolf) **tw：ran-huang**
```

Reply:
I think the [Line 47](https://github.com/pingcap/docs/pull/10249/files#diff-577672dd123c42d722caad525849ff7ee636eab7a8ca14597c7e674a93cbc04fR47) below has mentioned the support of `ALTER USER` statement. Do you think we should combine them in one description?

Reply:
Yes. However both options are fine with me.

Reply:
@ran-huang Please help combine the description of `CREATE USER` and `ALTER USER` statements into one. Thanks.